test/integration/concierge_impersonation_proxy_test.go

```diff
@@ -1793,6 +1797,10 @@ func TestImpersonationProxy(t *testing.T) { //nolint:gocyclo // yeah, it's compl
 	})
 
 	t.Run("using externally provided TLS serving cert with stringData", func(t *testing.T) {
+		if env.Proxy == "" {
+			t.Skip("Skipping ClusterIP test because squid proxy is not present")
```

Comment:
Note to reviewers: these tests messed with the CredentialIssuer's spec a lot. When they take away the annotation which tells GKE to only assign internal IPs to the load balancer, then the load balancer gets confused and stops working. To solve this, I'm skipping these tests except on Kind clusters, which still gives us good enough coverage for these behaviors.